testsuite/libra-fuzzer/src/fuzz_targets/transaction.rs

```diff
@@ -54,3 +60,82 @@ impl FuzzTargetImpl for SignedTransactionTarget {
         let _: Result<SignedTransaction, _> = lcs::from_bytes(&data);
     }
 }
+
+#[derive(Clone, Debug, Default)]
+pub struct MutatedSignedTransaction;
+
+static SIGNED_TXN: Lazy<SignedTransaction> = Lazy::new(|| {
+    let seed = [0u8; 32];
+    let recorder_rng = test_runner::TestRng::from_seed(RngAlgorithm::ChaCha, &seed);
+    let mut runner = TestRunner::new_with_rng(test_runner::Config::default(), recorder_rng);
+    SignedTransaction::arbitrary()
+        .new_tree(&mut runner)
+        .expect("creating a new value should succeed")
+        .current()
+});
+
+static SERIALIZED_SIGNED_TXN: Lazy<Vec<u8>> =
+    Lazy::new(|| lcs::to_bytes(&SIGNED_TXN.clone()).expect("serialization should work"));
+
+impl FuzzTargetImpl for MutatedSignedTransaction {
+    fn description(&self) -> &'static str {
+        "SignedTransaction (LCS serialized -> mutation -> deserializer)"
+    }
+
+    fn generate(&self, _idx: usize, _gen: &mut ValueGenerator) -> Option<Vec<u8>> {
+        println!("{:?}", SIGNED_TXN);
+        // always return the same serialized signed transaction
```

Comment:
this comment isn't that useful since it says what the code is doing in more words, so maybe we could state why we always returning the same signed transaction?